src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -251,154 +397,129 @@ export class ScriptEvaluator {
             executionContextId: realm.executionContextId,
           }
         );
-        // TODO (#375): release `argEvalResult.result.objectId` after using.
+        // TODO(#375): Release `argEvalResult.result.objectId` after using.
         return {objectId: argEvalResult.result.objectId};
       }
 
-      // TODO(#375): dispose of nested objects.
+      case 'channel': {
+        const channelName = argumentValue.value;
 
-      default:
-        throw new Error(
-          `Value ${JSON.stringify(argumentValue)} is not deserializable.`
+        const argEvalResult = await realm.cdpClient.sendCommand(
+          'Runtime.callFunctionOn',
+          {
+            functionDeclaration: String(() => {
+              const queue: string[] = [];
+              let resolver: (message: string | PromiseLike<string>) => void;
+
+              return {
+                /**
+                 * Gets a promise, which is resolved as soon as a message occurs
+                 * in the queue.
+                 */
+                async getMessage(): Promise<string> {
+                  if (queue.length > 0) {
+                    return queue.shift()!;
+                  }
+                  const promise = new Promise<string>((resolve) => {
+                    resolver = resolve;
+                  });
+                  promise.then(() => queue.shift());
+                  return promise;
+                },
```

Comment:
This change is required for option **1** _`sendMessage` should send message via `queue`_.
```suggestion
```

```suggestion
                async getMessage(): Promise<string> {
                  if (queue.length > 0) {
                    resolver = null;
                    return queue.shift()!;
                  }
                  const promise = new Promise<string>((resolve) => {
                    resolver = resolve;
                  });
                  promise.then(getMessage);
                  return promise;
                },
```